packages/website/src/theme/MDXComponents/RuleAttributes.tsx

```diff
@@ -6,12 +6,18 @@ import type { FeatureProps } from './Feature';
 import { Feature } from './Feature';
 import styles from './RuleAttributes.module.css';
 
-const getRecommendation = (docs: RuleMetaDataDocs): [string, string] => {
-  return docs.recommended === 'strict'
-    ? ['🔒', 'strict']
-    : docs.requiresTypeChecking
-    ? ['🧠', 'recommended-requiring-type-checking']
-    : ['✅', 'recommended'];
+const recommendations = {
+  recommended: ['✅', 'recommended'],
+  strict: ['🔒', 'strict'],
+  stylistic: ['🎨', 'stylistic'],
```

Comment:
Per https://github.com/bmish/eslint-doc-generator/pull/136 

cc @bmish - thought you'd enjoy this 😄